package.json

```
@@ -11,7 +11,8 @@
     "build": "[ \"$WATCH\" == 'true' ] && rollup -cw || rollup -c",
     "watch": "WATCH=true npm run build",
     "prebuild": "rm -rf dist",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
```

not really a part of your pr please forgive me
```suggestion
    "prepublishOnly": "npm run build",
```